extensions/blocks/videopress/deprecated/v2/index.js

```diff
@@ -59,17 +59,16 @@ export default {
 	support: {
 		reusable: false,
 	},
-	isEligible: ( { className, videoPressClassNames, guid } ) => {
+	isEligible: ( { videoPressClassNames, guid } ) => {
 		const missingClassNames = videoPressClassNames === undefined;
-		const missingVideoClass = ! className?.indexOf( 'wp-block-video' ) >= 0;
 
-		return guid && ( missingClassNames || missingVideoClass );
+		return guid && missingClassNames;
 	},
 	migrate: attributes => {
 		const { align, className, videoPressClassNames } = attributes;
 		return {
 			...attributes,
-			className: 'wp-block-video',
+			className: '',
```

Comment:
We will lose any custom classes a user added to block doing this, so probably need to add here some way of removing all the videopress classes from className at this point and just leave custom ones

Reply:
They would have been included into the `videoPressClassNames` attribute. That's how the block was previously working.

Reply:
Actually, after switching back to master and testing, master loses custom CSS classes.

Reply:
The migration of `className` to `videoPressClassNames` now takes into account custom CSS classes.